bitser.lua

```diff
@@ -434,6 +435,8 @@ end, loads = function(str)
 	end
 	Buffer_newReader(str)
 	return deserialize_value({})
+end, handleMetatables = function(bool)
```

Comment:
A bit of a nitpick, but I would call it `includeMetatables` or something like that.

Reply:
Yep, sure.